waymo_raw_data_to_argoverse.py

```diff
@@ -216,19 +258,89 @@ def dump_pose(city_SE3_egovehicle, timestamp, log_id):
 		'rotation': [w, x, y, z],
 		'translation': [x,y,z]
 	}
-	json_fpath = f'TEST_RAW_DATA/pose_logs/{log_id}/poses/city_SE3_egovehicle_{timestamp}.json'
+	json_fpath = f'{parent_path}/{log_id}/poses/city_SE3_egovehicle_{timestamp}.json'
 	check_mkdir(str(Path(json_fpath).parent))
 	save_json_dict(json_fpath, pose_dict)
 
+def dump_point_cloud(points, timestamp, log_id, parent_path):
+	data = {'x': points[:,0], 'y': points[:,1], 'z': points[:,2]}
+	cloud = PyntCloud(pd.DataFrame(data))
+	cloud_fpath = f'{parent_path}/{log_id}/lidar/PC_{timestamp}.ply'
+	check_mkdir(str(Path(cloud_fpath).parent))
+	cloud.to_file(cloud_fpath)
+
+def dump_object_labels(labels, timestamp, log_id, parent_path):
+	argoverse_labels = []
+	for label in labels:
+		argoverse_labels.append(build_label(label, timestamp))
+	json_fpath = f'{parent_path}/{log_id}/per_sweep_annotations_amodal/tracked_object_labels_{timestamp}.json'
+	check_mkdir(str(Path(json_fpath).parent))
+	save_json_dict(json_fpath, argoverse_labels)
+
+def build_label(label, timestamp):
```

Comment:
maybe we could name this `build_argo_label()` or `convert_waymo_to_argo_label()` to be a bit more descriptive?

Reply:
Addressed in [this commit](https://github.com/johnwlambert/waymo_to_argoverse/pull/2/commits/8931614c713c462a1efbbc0f06a366d44a28df25)